frontend/src/Components/Player.jsx

```diff
@@ -20,7 +20,6 @@ export default function Player({ playerId, playerName, style, childRef }) {
         [styles.isHoverable]: isHoverable,
         [styles.hasVoted]: hasVoted,
         [styles.isClicked]: isClicked,
-        [styles.isDead]: isDead,
```

Comment:
We still need these line to style dead players right?

Reply:
Yes. Indeed. I believe I accidentally removed this line of code ...